### PR TITLE
Edited the Opacity level

### LINKED ISSUE #137270: The foreground here is very dark, hope we can allow us to modify it ourselves
Hello, great teacher!
        In the long-term use of vscode, I found that the number of method parameter overloads has a dark foreground color, and my eyes will be tired if I see it for a long time. I ask the teacher to allow us to modify the foreground color to our own suitable foreground color. It will be great for people who have been coding for a long time! Hope this problem can get your support and help from the teacher to improve it in the next milestone. Thank you very much!
方法参数前景色很黑暗

### PR DESCRIPTION
**Pull request by ghost**
This PR fixes #137270

### CODE REVIEW

File: src/vs/editor/contrib/parameterHints/parameterHints.css [hunk 1]
```diff
@@ -96,7 +96,7 @@
 	text-align: center;
 	height: 12px;
 	line-height: 12px;
-	opacity: 0.5;
+	opacity: 0.1;
```
Comment (gjsjohnmurray) [inline src/vs/editor/contrib/parameterHints/parameterHints.css:99]: My understanding of the issue is that the opacity needs to be increased, not decreased.

Comment (najmieh) [reply src/vs/editor/contrib/parameterHints/parameterHints.css]: Thank you for the feedback.
Yes, I changed that.

**Comment by misolori** [comment]
I'd maybe remove the opacity all together as we're working on removing this from our code base because of accessibility issues. Here's what it looks like without any opacity tweaks:

CleanShot 2022-01-28 at 13 32 33@2x

**Comment by mjbvz** [comment]
Closing this since the user seems to have deleted their account

**Pull request closed without merging**